src/components/RightSidebar/TableOfContents.tsx

```diff
@@ -90,16 +94,18 @@ const TableOfContents: FunctionComponent<Props> = ({ toc = [], labels, isMobile
 		return () => headingsObserver.disconnect();
 	}, []);
 
-	const onLinkClick = (e) => {
+	const onLinkClick = (e: MouseEvent) => {
```

Comment:
Similar here and below:

```suggestion
	const onLinkClick = (e: JSX.TargetedMouseEvent<HTMLAnchorElement>) => {
```